tests/test_constraints.py

```diff
@@ -36,6 +36,27 @@ def test_generate_cell_methods_constraint():
     assert expected_cell_methods_constraint in repr(cell_methods_constraint)
 
 
+def test_generate_time_constraint():
+    """generate iris cube constraint for dates."""
+    from datetime import datetime
```

Comment:
does PEP8 not enforce imports at the top of script (not sure if black checks this?)

Reply:
Fixed in 051060bfa0f612cbad88f5fc84bbd7ebad14acb3.